pycg/machinery/definitions.py

```diff
@@ -199,50 +205,49 @@ class Definition:
         utils.constants.EXT_DEF,
     ]
 
-    def __init__(self, fullns, def_type):
+    def __init__(self, fullns: str, def_type) -> None:
         self.fullns = fullns
-        self.points_to = {
```

Comment:
This bothered me quite a bit that a dictionary was getting created for every definition just to hold two different variables. Then the dictionary got in the way of correctly inferring the types of the two kinds of pointers and this is all cleaner now.